deep_quoridor/src/agents/alphazero/mcts.py

```diff
@@ -106,8 +106,8 @@ def backpropagate_result(self, value: float):
         """
         self.value_sum += value
         self.visit_count += 1
-        self.wins = self.wins + 1 if value == 1 else 0
-        self.losses = self.losses + 1 if value == -1 else 0
+        self.wins = self.wins + (1 if value == 1 else 0)
+        self.losses = self.losses + (1 if value == -1 else 0)
```

Comment:
this would was resetting the wins and losses rather than not incrementing the value